README.md

```diff
@@ -114,6 +114,11 @@ From this point on, we'll assume that your web server is available at this local
 
 http://localhost:3000
 
+Author API : http://localhost:3000/author/index
+Questions API : http://localhost:3000/questions/index
+Items API : http://localhost:3000/items/index
+Reports API : http://localhost:3000/reports/index
+
```

Comment:
@bhavya-shukla-lrn we should perhaps update the following line of documentation, since you have now added many more examples of functionality. We could replace the following line here (122) with this:

"Open these pages with your web browser. These are all basic examples of Learnosity's integration. You can interact with these demo pages to try out the various APIs. The Items API example is a basic example of an assessment loaded into a web page with Learnosity's assessment player. You can interact with this demo assessment to try out the various Question types."